src/OSPSuite.Core/Comparison/ExpressionProfileDiffBuilder.cs

```diff
@@ -12,7 +13,7 @@ public override void Compare(IComparison<ExpressionProfileBuildingBlock> compari
       {
          base.Compare(comparison);
          CompareValues(x => x.Type, x => x.Type, comparison);
-         _enumerableComparer.CompareEnumerables(comparison, x => x.InitialConditions, x => x.Path);
```

Comment:
I am not sure why this was removed? It makes sense for me to have a property InitialCoditions get at the ExpressionProfile level. Using the expression profile as an IEnumerable InitialCondition is not *intuitive*